cns/azure-cns.yaml

```diff
@@ -103,6 +103,10 @@ spec:
               mountPath: /var/lib/azure-network
             - name: cns-config
               mountPath: /etc/azure-cns
+            - name: azure-vnet
+              mountPath: /var/run/azure-vnet
```

Comment:
@rbtr What was `/var/run/azure-vnet` used for in CNS? Just trying to map these to windows

Reply:
cni state and lock file